titan-web-client/src/layouts/dashboard/index.js

```diff
@@ -5,6 +5,7 @@ import Sidebar from './components/sidebar/Sidebar'
 
 export const DashboardWrapper = styled.div`
   width: 100%;
+  height: 100vh;

```

Comment:
All changes to `DashboardWrapper` fix a column alignment bug in the dashboard layout when content is too short or too long.